spec/features/reviews/03_update_reviews_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
```

Comment:
Use `let` statements to DRY up test code when you have variables, etc that you are sharing between your different scenarios:

```
feature "..." do
  let!(:review) { create(:review) }
  let(:park) { review.park }
  let(:user) { review.user }

  scenario "review creator ..." do
    sign_in_as user
    ...
  end

  scenario "review creator creates ..." do
    sign_in_as user
    ...
  end
end
```

And in this case, you'd want to use `let!` for `create(:review)`, rather than `let`, since you *need* to have that created before it's referenced within the scenario (i.e. using `let` would save its creation until you call `click_link(park.title)`, and as it hasn't been created yet there wouldn't be a link. `let!` ensures that it's created before the scenario runs)